src/typescript/azure-openapi-validator/rules/LintDescriptionProse.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const LintDescriptionProse: string = "LintDescriptionProse";
```

Comment:
this rules appears that could expose a lot of violations, I worry about noise, not having a way to filter doc violations.

Reply:
Please see the comment associated with the PR. When additional rules are added, I would like to spec in `x-ms-disable-lint` as a Swagger extension which lists rules to ignore by ID.